blog/2021-06-01-teraspotlight-gsoc-2021-projects/index.md

```diff
@@ -224,7 +224,7 @@ This will serve as the foundation for more interesting generators, allowing for
 The project will be mentored by _@Nicholas Bates_ and _@Cervator_, with additional support from last year's student _@Isaac L_.
 
   <div align="center">
-    <img src="./dest-sol-worldgen.png" />
```

Comment:
lets keep this as PNG to preserve transparency (they are roughly the same size or even smaller anyways).